internal/server/options.go

```diff
@@ -1,17 +1,104 @@
 package server
 
-func WithSessionName(name string) func(*Server) {
-	return func(s *Server) {
+import (
+	"encoding/base64"
+	"io/ioutil"
```

Comment:
io/ioutil was deprecated in go 1.16. All of the functionality was moved to the io and os so references to ioutil should be refactored to use the appropriate replacements listed in https://golang.org/doc/go1.16#ioutil.

Reply:
Good catch. Fixed, thanks!

Reply:
Indeed.  I wonder if our linting golangci needs a version bump.  I feel like it should catch stuff like this.